src/app/ApplicationTemplate.Client/ApiEndpoints/Authentication/AuthenticationToken.cs

```diff
@@ -4,9 +4,9 @@
 
 namespace ApplicationTemplate.Client;
 
-public record AuthenticationToken
+public class AuthenticationToken
 {
-	public static AuthenticationToken Default { get; } = new AuthenticationToken(default, DateTimeOffset.MinValue, DateTimeOffset.MinValue);
+	public AuthenticationToken() { }
```

Comment:
Should we be able to create an empty token?

Reply:
reverted this part. Sorry about that, it was part of my investigation